datalad/support/gitrepo.py

```diff
@@ -881,24 +1091,17 @@ def clone(cls, url, path, *args, clone_options=None, **kwargs):
         for trial in range(ntries):
             try:
                 lgr.debug("Git clone from {0} to {1}".format(url, path))
-                with GitPythonProgressBar("Cloning") as git_progress:
-                    repo = gitpy.Repo.clone_from(
-                        url, path,
-                        env=env,
-                        # we accept a plain dict with options, and not a gitpy
-                        # tailored list of "multi options" to make a future
-                        # non-GitPy based implementation easier. Do conversion
-                        # here
-                        multi_options=to_options(**clone_options) if clone_options else None,
-                        odbt=default_git_odbt,
-                        progress=git_progress
-                    )
-                # Note/TODO: signature for clone from:
-                # (url, to_path, progress=None, env=None, **kwargs)
 
+                # TODO bring back progress reporting
+                with GitProgress() as progress:
+                    WitlessRunner(env=env).run(
```

Comment:
Hmm, looking at the previous code, I guess this keeps the same `env`.  I'm a bit confused because I would think that it should pass the same environment that `GitRunner.run` does (i.e., one adjusted with `get_git_environ_adjusted`).

Reply:
True. I added an appropriate TODO to remove this from GitRepo entirely, and for now converted this one case.

Reply:
Removed in full in my local branch of #4087